sdcm/cluster.py

```diff
@@ -1186,7 +1189,11 @@ def node_setup(node):
             node.remoter.run('sudo yum install -y screen')
             if db_node_address is not None:
                 node.remoter.run("echo 'export DB_ADDRESS=%s' >> $HOME/.bashrc" %
-                                 db_node_address)
+                                     db_node_address)
```

Comment:
This small change makes the CI job to be unhappy. remember to use inspektor (`pip install inspektor`, `inspektor style --fix`) when writing the patches.